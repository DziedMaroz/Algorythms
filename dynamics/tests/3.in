14269581424
453
18451787
13715221
0

747921
6821
840
25
72670
7
21405
099

7
8900797
10834
03125242
7359474302
4554386682
79287381



9197
100120184
2933074299
16864
06665559

4
901472038
395620901
23036359

8167325
081604598
729339
8097850
13
9245422
47016236


0958
580032
750
7181860762
952
205
16545
104969719
491346
299822
234404
24349

393687116
01
06914803
21869282
926

7910222987
186784636
42531
0220
0794903
6797606398
55851274

629001930
39976198
0138585

841444453

11
7515223118
7
789428
1311467
73568144
67970250
196214568
832
591735873

623463
415718288
71085
04
1527384099
996408
6513678087
35203054
13208567
621
46
21
032084
962

447
8296011936
489
7574
117256
0954247227
501123
3560370599
00416112
8356864649
62
766
17807932
785
407
0888531125

7
5069575
9977
227401
485370024
603
14
8
52917562
33086274
34417
39607838
969871
619003285
89

35
05788
68933951
4496353
346989
56589
930
23129

984364605
404839
2931516959
71494


8040257
683223552
1450
76
9665910
165
32
6340

393785448
38056136
42552917
0159307
9844
8866

0196
80746667
78188
9
1
05574307
75645
4651
4
7597814010
092938

124749698
82570
66
59057
394569994
75920
48390461

74956204
980348

66
6351
340
426269224
186206151
697005893
19658488
821833
67079
4584452
50285834
8281219309
69722715
403897
73562132
180
603390
41135
2648118831
51949773
86
7789777513
366485
766
600430630
5
859410836
118338957
7408
25637
33
759
3332225072
60209348
1
66704
37549761
1

39268284
085067

86
639352540
87777
37153
188619948
726
1
02588337
2
7
457
84
0557
634130824
146462
2791421709
411590507
5437045752
9697874
90563489
386695752
93
0982276632
3830373
1624359516

660306
26414
4351370451
49
145
912468518
344420364
1661334931
90129213
88208
409
75
165886876
9258606626
60
559028755
1403
18121281
40998
518
083758
628
6

035
46151064
11
56
441
8

3382
56453
516
839578088
3
0290344

118
0
93624699
1
3998291
1599
56
608051
04
20
935672
3326
9731472
42046326
7075
562
49
89987
7046203420
483
3418
391251951
4605385518
7
094698251
679881983
05679
9412785145
44755
022
812132
2
66875
827926332
2693
29553091
1

7
8693
472250
802264
331366
85
8860
100797
085266
998415147
0657870636
298031
9206
4996
842491659
611988
7155576019
7
929080
349
7008
878505
3327077
30792
6304497
11
125
738065
929434379
1939610
188
9
0529345553
82100
1608043
46658
48927237
163
40051840
44
4645326358
07729
39259
9031023721
89
548
2
83573826
83
0

11208
7277717
72
86828779

9
41
369

43163
4
931
573
326407839
0
47897030
6111533
0035514
496
0367
81634
13707
7118685
27
2
477618619
834

902637
3116791

7103234
69254890
0260
98011733
674622
9081043
3723437789
903780519
23348919
7366
7849590406
6354521
315965123
778
33811
363976
982
259467981
21015766
95
275

830258
7274
9049197
11
114348

7
89
5568150553
6008376
4166053
779102507
9462105474
89
00110143
4279790529
715400395
