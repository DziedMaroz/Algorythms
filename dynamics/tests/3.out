8
14 2 6 95 8 14 2 4
